mix.exs

```diff
@@ -30,8 +30,8 @@ defmodule Intercom.Mixfile do
   # Type "mix help deps" for more examples and options
   defp deps do
     [
-      {:estree, "~> 2.3.0"},
-      {:httpoison, "~> 0.8"}
+      {:estree, "~> 2.4.2"},
```

Comment:
@willrax can you update to 2.5? And maybe also remove the patch version here? 

This way we would not need to update the dependencies version here every minor release. And I guess we would not have problems since we use very basic features of this dependencies, if they change these features, they would rather change the major release version.